lib/times-table.js

```diff
@@ -0,0 +1,35 @@
+var pry = require('pryjs');
+
+function createArray(number){
+  newArray = []
```

Comment:
you're creating a great deal of global variables here - make sure you're creating local variables instead. we want to use the `var` keyword for this in ES5 or the `let` keyword in ES6. 

so here, you'd have `var newArray = []`